재귀로 풀지 않으면 무조건 런타임 에러가 발생했다.
input = sys.stdin을 사용할 경우에도 마찬가지였다.
